DockerBuildManagement/ChangelogSelections.py

```diff
@@ -26,7 +30,9 @@ def ExportChangelogSelection(changelogSelection):
         return
 
     VersionTools.ExportVersionFromChangelogToEnvironment(
-        changelogSelection[FILE_KEY], YamlTools.TryGetFromDictionary(changelogSelection, ENV_VERSION_KEY, 'VERSION'))
+        changelogSelection[FILE_KEY], YamlTools.TryGetFromDictionary(changelogSelection, ENV_VERSION_KEY, 'VERSION'),
+        YamlTools.TryGetFromDictionary(changelogSelection, ENV_VERSION_MAJOR_KEY, None),
```

Comment:
Maybe just expose a default envMajorVersionKey/envMinorVersionKey? Easier to use maybe. Default then as VERSIONMAJOR/VERSIONMINOR

Reply:
I will leave the default as None for now as discussed, since versionmajor/versionminor will not be usefull in all scenarios.